llvm/include/llvm/Transforms/Utils/Local.h

```diff
@@ -478,6 +479,10 @@ void hoistAllInstructionsInto(BasicBlock *DomBlock, Instruction *InsertPt,
 DIExpression *getExpressionForConstant(DIBuilder &DIB, const Constant &C,
                                        Type &Ty);
 
+/// Remap the debug intrinsic instructions in the \p Mapping using the
+/// \p Inst as a criteria.
```

Comment:
I think the wording of this comment is slightly unclear. What do you think of this instead?

"Remap the operands of the debug records attached to \p Inst, and the operands of \p Inst itself if it's a debug intrinsic."